sound/soc/sof/intel/hda-dai-ops.c

```diff
@@ -416,10 +416,11 @@ static int hda_ipc4_post_trigger(struct snd_sof_dev *sdev, struct snd_soc_dai *c
 		break;
 	case SNDRV_PCM_TRIGGER_SUSPEND:
 	case SNDRV_PCM_TRIGGER_STOP:
-		/*
-		 * STOP/SUSPEND trigger is invoked only once when all users of this pipeline have
-		 * been stopped. So, clear the started_count so that the pipeline can be reset
-		 */
```

Comment:
This comment still stands as we clear the started_count, should we keep it?

Reply:
@ujfalusi the comment was useful before because we set the started_count to 0 before the pipeline was reset in the FE DAI hw_free. But now the pipeline has just been reset right here in this op, so it is only natural to clear the started_count.